Vagrantfile

```diff
@@ -2,27 +2,33 @@
 Vagrant::configure("2") do |config|
 
   # configure the basebox
-  config.vm.box = "tknerr/ubuntu1604-desktop"
-  config.vm.box_version = "2.0.27.1"
+  config.vm.box = "fasmat/ubuntu1804-desktop"
```

Comment:
Sweet, that's how you end up with your own basebox! :)

I found https://app.vagrantup.com/fasmat/boxes/ubuntu1804-desktop here, but was wondering how you built that basebox in the first place. In case it was build using packer, a reference to the repo with the packer template that was used to create the box would be helpful (in the basebox description on app.vagrantup.com I mean).

My https://app.vagrantup.com/tknerr/boxes/ubuntu1604-desktop for example was based on https://github.com/boxcutter/ubuntu.

Happy to use yours, was just wondering how to find out how it was built and what's included

Reply:
I forked boxcutter/ubuntu and updated it to be able to create a 18.04 image. I thought I had the link in the box description, but thanks for the hint! I'll add it tomorrow.

See the following pull request for the changes I made to be able to create the base box:
https://github.com/boxcutter/ubuntu/pull/162

Reply:
Sweet, thanks for the info! 👍